src/Provider/PrefillServiceProvider.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Bolt\Provider;
+
+use Silex\Application;
+use Silex\ServiceProviderInterface;
+use Bolt\Storage\Prefill;
+
+class PrefillServiceProvider implements ServiceProviderInterface
+{
+    public function register(Application $app)
+    {
+        $app['prefill'] = $app->share(
```

Comment:
I know this to some extent belongs in our discussion about the 2.2 refactor, but while adding a `Bolt\Storage` namespace, should we also namespace the services, i.e. `$app['storage.prefill']`?